test/integration/targets/ansible-galaxy-role/tasks/main.yml

```diff
@@ -56,3 +56,14 @@
 
 - name: Uninstall invalid role
   command: ansible-galaxy role remove invalid-testrole
+
+- name: Install invalid role from remote requirements.yml
+  command: ansible-galaxy install -r https://github.com/eryx12o45/ansible/tree/add_new_galaxy_option/test/integration/targets/ansible-galaxy-role/files/invalid-requirements.yml
```

Comment:
This would need to be a URL to some of the repositories controlled by us. We cannot rely on arbitrary content on the internet.

Reply:
Sure thing, just for the imitial testing it had to be a valid url. I can change this nevertheless once there is a valid file